code/datums/gamemode/role/cultist.dm

```diff
@@ -0,0 +1,34 @@
+/datum/role/cultist
+	name = "Cultist"
+	protected_jobs = list("Security Officer", "Warden", "Detective", "Head of Security", "Captain", "Chaplain")
```

Comment:
That's a balance question I need to ponder upon. HoP roundstart cultists are a thing currently. I might remove it since it gives the cult too much of an edge early.